<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen</class>
 <widget class="QWidget" name="Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelPictrue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarHP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelHP">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarES">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>95</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelES">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>95</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
